### PR TITLE
BitRecover.WindowsLiveMailConverter version 7.5

### PR DESCRIPTION
**Pull request by OyePradumya**
- [ ] Have you signed the [Contributor License Agreement](https://cla.opensource.microsoft.com/microsoft/winget-pkgs)?
- [ ] Have you checked that there aren't other open [pull requests](https://github.com/microsoft/winget-pkgs/pulls) for the same manifest update/change?
- [ ] Have you [validated](https://github.com/microsoft/winget-pkgs/blob/master/AUTHORING_MANIFESTS.md#validation) your manifest locally with `winget validate --manifest <path>`? 
- [ ] Have you tested your manifest locally with `winget install --manifest <path>`?
- [ ] Does your manifest conform to the [1.1 schema](https://github.com/microsoft/winget-pkgs/tree/master/doc/manifest/schema/1.1.0)?

Note: `<path>` is the name of the directory containing the manifest you're submitting.

-----

###### Microsoft Reviewers: [Open in CodeFlow](https://portal.fabricbot.ms/api/codeflow?pullrequest=https://github.com/microsoft/winget-pkgs/pull/59416)

### CODE REVIEW

**Review by OyePradumya — COMMENTED** [review]
Made changes with the latest hash information

File: manifests/b/BitRecover/WindowsLiveMailConverter/7.0/BitRecover.WindowsLiveMailConverter.installer.yaml [hunk 1]
```diff
@@ -0,0 +1,13 @@
+# Created using wingetcreate 1.0.3.0
+# yaml-language-server: $schema=https://aka.ms/winget-manifest.installer.1.1.0.schema.json
+
+PackageIdentifier: BitRecover.WindowsLiveMailConverter
+PackageVersion: 7.0
```
Comment (ItzLevvie) [inline manifests/b/BitRecover/WindowsLiveMailConverter/7.0/BitRecover.WindowsLiveMailConverter.installer.yaml:5]:
```suggestion
PackageVersion: 7.5
```

File: manifests/b/BitRecover/WindowsLiveMailConverter/7.0/BitRecover.WindowsLiveMailConverter.yaml [hunk 2]
```diff
@@ -0,0 +1,9 @@
+# Created using wingetcreate 1.0.3.0
+# yaml-language-server: $schema=https://aka.ms/winget-manifest.version.1.1.0.schema.json
+
+PackageIdentifier: BitRecover.WindowsLiveMailConverter
+PackageVersion: 7.0
```
Comment (ItzLevvie) [inline manifests/b/BitRecover/WindowsLiveMailConverter/7.0/BitRecover.WindowsLiveMailConverter.yaml:5]:
```suggestion
PackageVersion: 7.5
```

Comment (ItzLevvie) [reply manifests/b/BitRecover/WindowsLiveMailConverter/7.0/BitRecover.WindowsLiveMailConverter.yaml:5]: image

File: manifests/b/BitRecover/WindowsLiveMailConverter/7.0/BitRecover.WindowsLiveMailConverter.installer.yaml (hunk 1 above)
Comment (ItzLevvie) [reply manifests/b/BitRecover/WindowsLiveMailConverter/7.0/BitRecover.WindowsLiveMailConverter.installer.yaml:5]: image

File: manifests/b/BitRecover/WindowsLiveMailConverter/7.0/BitRecover.WindowsLiveMailConverter.locale.en-US.yaml [hunk 3]
```diff
@@ -0,0 +1,26 @@
+# Created using wingetcreate 1.0.3.0
+# yaml-language-server: $schema=https://aka.ms/winget-manifest.defaultLocale.1.1.0.schema.json
+
+PackageIdentifier: BitRecover.WindowsLiveMailConverter
+PackageVersion: 7.0
```
[the remark this answers is not in the document]
Comment (ItzLevvie) [reply manifests/b/BitRecover/WindowsLiveMailConverter/7.0/BitRecover.WindowsLiveMailConverter.locale.en-US.yaml:5]: image

**Review by ItzLevvie — CHANGES_REQUESTED** [review]

File: manifests/b/BitRecover/WindowsLiveMailConverter/7.0/BitRecover.WindowsLiveMailConverter.locale.en-US.yaml [hunk 4]
```diff
@@ -0,0 +1,26 @@
...
+ShortDescription: This program gives the facility to migrate from WLM to multiple generic file extensions such as PST, MSG, EML, EMLX, TXT, MBOX, HTML, MHT, XPS, RTF, DOC, Windows Live Mail, CSV, Gmail, Yahoo, TIFF, BMP, GIF, JPEG, PNG, PDF, NSF, TGZ, OLM, etc.
+Description: This program gives the facility to migrate from WLM to multiple generic file extensions such as PST, MSG, EML, EMLX, TXT, MBOX, HTML, MHT, XPS, RTF, DOC, Windows Live Mail, CSV, Gmail, Yahoo, TIFF, BMP, GIF, JPEG, PNG, PDF, NSF, TGZ, OLM, etc. It gives freedom to transfer folders from Windows Live Mail to various cloud-based email services and servers, i.e., Gmail, Yahoo, Outlook.com, Office 365, Exchange Server, Hosted Exchange Server, G Suite, IBM Verse, Amazon WorkMail, IMAP Server, etc. The tool has an inbuilt feature to automatically load WLM folders in the software panel from the default storage location. It has another option to load emails manually from any other storage location. Moreover, after the start, the tool is will automatically load overall Windows Live Mail databases in the software panel, including all email folders in Windows Explorer style.
+Moniker: bitrecover
```
Comment (ItzLevvie) [inline manifests/b/BitRecover/WindowsLiveMailConverter/7.0/BitRecover.WindowsLiveMailConverter.locale.en-US.yaml:17]:
```suggestion
Moniker: windows-live-mail-converter
```

**Review by ItzLevvie — CHANGES_REQUESTED** [review]

**Review by ItzLevvie — CHANGES_REQUESTED** [review]
You'll also need to update the folder version to `7.5`

**Review by ItzLevvie — APPROVED** [review]

**Pull request merged**